packages/gatsby-theme-carbon/src/styles/internal/_page.scss

```diff
@@ -134,6 +134,15 @@ img {
   padding-right: 0;
 }
 
+.container .#{$prefix}--no-gutter-sm--yes-gutter-lg {
```

Comment:
Adding rules to the _page should be avoided at all costs, this stylesheet has in the past become a sort of dumping ground for global rules. 

Like I mentioned in the earlier comment you can imagine things getting spicy if we keep trying to put behavior like this into sass.

e.g. `.#{$prefix}--no-gutter-sm--yes--gutter-lg--no-gutter-max`

Reply:
Totally get it. I felt gross when i typed it ;)